test/OpenSpaceToolkit/Astrodynamics/Trajectory/Validation/NRLMSIS00.validation.cpp

```diff
@@ -0,0 +1,327 @@
+/// Apache License 2.0
```

Comment:
not really applicable to this file, but to its parent folder name `Validation`
should it be lower case (i.e. `validation`) ?

Reply:
Hmmm yeah idk what the best name for this is since it doesn't represent an actual OSTk class. Ideas?